doc/CodeReviews.md

```diff
@@ -0,0 +1,48 @@
+# Code Reviews
+
+Every GitHub pull request must go through a code review and approved before it will be merged into the master branch.
+
+## Assigning a Pull Request
+
+If you want to address your review to a particular set of teammates, add them as Assignee (righthand side on the pull request).
+They'll receive an email.
+
+During discussions, you can also refer to somebody using the *@username* syntax and they'll receive an email as well.
+
+If you want to receive notifications even when you aren't mentioned, you can go to the [repository page](https://github.com/youtube/vitess) and click *Watch*.
+
+## Approving a Pull Request
+
+As a reviewer you can approve a pull request through two ways:
+
+* Approve the pull request via GitHub's new code review system
+* reply with a comment that contains the word *LGTM*  (Looks Good To Me)
+
+## Merging a Pull Request
+
+Pull requests can be merged after they were approved and the Travis tests have passed.
+External contributions will be merged a team member.
```

Comment:
Fixed in commit 9aa199ceaff57e465cf406fe0f06ce5683372c6a and published.

Thanks for proof reading :)